### PR TITLE
[#2226] Hide inactive deployment flow sections.

### LINKED ISSUE #2226: Google Compute Engine input area is greyed out. There should be nothing showed by default
screen shot 2016-11-21 at 10 40 52

### PR DESCRIPTION
**Pull request by kadams54**
Fixes #2226.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6263/
Test PASSed.

File: jujugui/static/gui/src/app/components/expanding-row/expanding-row.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ YUI.add('expanding-row', function() {
     _resize: function() {
       const expanded = this.state.expanded;
       this.setState({styles: {
-        height: expanded ?  this.refs.inner.offsetHeight + 'px' : '0px',
+        height: expanded ?  'auto' : '0px',
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/expanding-row/expanding-row.js]: I believe this change will break the css animations. Lets chat about why this change was necessary.

**Comment by anthonydillon** [comment]
Code looks good but finding it hard to QA as:
1. Connecting to local JIMM is not working at the moment
2. guiproxy is already bootstrapped on a cloud

I dont know any other way to test this.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6556/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6568/

**Comment by kadams54** [comment]
## QA

1. Register (if you haven't already) the staging JIMM controller: `juju register jimm.staging.jujucharms.com:443`.
2. Fire up the GUI and guiproxy. You may want to fetch @frankban's latest on guiproxy, as he fixed a situation that makes QAing this easier.
3. Depending on whether you have any models already on the staging JIMM, you may need to either create a new one or just visit localhost:8042. Either way, the QA should happen in a new/uncommitted canvas.
4. Add a charm to the canvas.
5. Deploy.
6. Verify that the only panel visible is the one with the AWS and GCE logos. Once you select one, the appropriate panel should slide into view. Verify that both AWS and GCE panels appear.

No need to go any further in the DF process (i.e., actually spin up servers).

**Comment by kadams54** [comment]
Going to go ahead and :shipit: as the sections @hatched commented on have been removed. Thanks all!

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge
